editor/components/post-featured-image/index.js

```diff
@@ -30,12 +26,12 @@ function PostFeaturedImage( { featuredImageId, onUpdateImage, onRemoveImage, med
 			<div className="editor-post-featured-image">
 				{ !! featuredImageId &&
 					<MediaUpload
-						title={ postLabel.set_featured_image }
+						title={ __( 'Set featured image' ) }
```

Comment:
Any reason to avoid the labels from the post object `postLabel.set_featured_image` and `postLabel.remove_featured_image`. These labels are tweakable server-side. We could use the proposed labels as a fallback if the `set_featured_image` or `remove_featured_image` are not defined?

Reply:
I've no strong opinion here. In this case I changed it so it was consistent with the other labels.

"Set featured image" is the only label we haven't customized. If you'd like, I can restore all instances where we use "Set featured image" as the label to use the postLabel again.

Reply:
I feel we should restore it essentially for Backwards Compatibility because CPT are probably customizing this label. And I feel we consistently use CPT's labels across the app. In which components are we avoiding those?

Reply:
I have no moral objections to reapplying those labels. But here's what's in master:

```
const DEFAULT_SET_FEATURE_IMAGE_LABEL = __( 'Set featured image' );
const DEFAULT_REMOVE_FEATURE_IMAGE_LABEL = __( 'Remove featured image' );
```

Those labels match the ones from `postLabel`.

If this branch gets merged sans changes, then we will have the following labels:

- `__( 'Set featured image' )` — this could match the postLabel
- `__( 'Replace Image' )` — no match in postLabel
- `__( 'Remove Image' )` — could match "Remove featured image", but it feels super redundant and would take up more space.

The postLabels are definitely designed for the old interface:

<img width="306" alt="screen shot 2018-06-08 at 10 47 11" src="https://user-images.githubusercontent.com/1204802/41148554-86ae441e-6b09-11e8-9b68-4fe7e88f3045.png">

If we can update the postLabel labels, that would be ideal, but I imagine that's not possible. 

What do you think we should do?

Reply:
I personally don't think the labels for the featured images are that important because it's a generic thing:

 - We can try to add the `replace_featured_image` label to the CPT labels. (Not certain how we define the existence of these labels)
 - I'm also fine considering the featured image generic enough to avoid these labels but maybe we can ask someone more familiar with the plugins to know whether these are heavily used or not? I don't feel confident making the decision myself.

Reply:
Solid thoughts.

@pento, any idea here?

Reply:
@afercia you also have a treasure trove of core experience — do you have any insights here?

Reply:
@jasmussen sorry, I'm late to the party. Is your question related to how many plugins might use custom labels? Not a great plugins expert here, but I guess it's pretty common for plugins to customize them, just a couple examples: EDD and Woocommerce:
```
Easy digital downloads:
'featured_image'        => __( '%1$s Image', 'easy-digital-downloads' ),
'set_featured_image'    => __( 'Set %1$s Image', 'easy-digital-downloads' ),
'remove_featured_image' => __( 'Remove %1$s Image', 'easy-digital-downloads' ),
'use_featured_image'    => __( 'Use as %1$s Image', 'easy-digital-downloads' ),

Woocommerce:
'featured_image'        => __( 'Product image', 'woocommerce' ),
'set_featured_image'    => __( 'Set product image', 'woocommerce' ),
'remove_featured_image' => __( 'Remove product image', 'woocommerce' ),
'use_featured_image'    => __( 'Use as product image', 'woocommerce' ),
```